src/SearchBar.js

```diff
@@ -58,6 +68,12 @@ class SearchBar extends React.Component {
           </Form>
         </div>
 
+        {
+          <Alert show={this.state.setShow} variant="danger">
+            <Alert.Heading>An error has occurred!</Alert.Heading>
+            <p>{this.state.errorOccur[1]}</p>
+          </Alert>
+        }
```

Comment:
I'm a little unclear on this part. To make sure I'm understanding, I think this refers back to state up on top of the page.